packages/algob/src/lib/tx.ts

```diff
@@ -247,3 +247,88 @@ export async function transferASALsig (
 
   return await deployer.waitForConfirmation(tx1.txId);
 }
+
+export async function callNoOpSSC (
+  deployer: AlgobDeployer,
+  sender: AccountSDK,
+  payFlags: TxParams,
+  index: number,
+  appArgs?: Uint8Array[],
+  accounts?: string,
+  foreignApps?: string,
+  foreignAssets?: string,
+  note?: Uint8Array,
+  lease?: Uint8Array,
+  rekeyTo?: string
+): Promise<void> {
+  const params = await mkSuggestedParams(deployer.algodClient, payFlags);
+  const txn = tx.makeApplicationNoOpTxn(
+    sender,
+    params,
+    index,
```

Comment:
(for all functions) can we change this to `appId` for more clarity? If not, then please add a small comment `index: the app-id as defined by the create method result`.

Reply:
yes